server/db/db.js

```diff
@@ -325,6 +330,64 @@ module.exports.getInstallationWithInstallationId = async function(installationId
     return null
 }
 
+module.exports.getHeartbeats = async function(client) {
+    let transactionMode = (typeof client !== 'undefined')
+    if(!transactionMode) {
+        client = await pool.connect()
+    }
+    
+    const { rows } = await client.query("SELECT * FROM heartbeat")
```

Comment:
Also, what do you think about adding `ORDER BY system_name` to the end of this query? I think it would be nicer for when it's displayed on the dashboard to keep it in a fixed order.